coffea/processor/executor.py

```diff
@@ -1823,12 +1832,15 @@ def run(
 
         executor = self.executor.copy(**exe_args)
 
-        wrapped_out, e = executor(chunks, closure, None)
-        if wrapped_out is None:
+        if chunks != []:
```

Comment:
This could be very expensive if chunks is not small or empty. Change this to len(chunks) > 0?

Reply:
changed to `len(chunks)!=0` now. Sorry for the delay I had missed this comment :)